File: function.js
// function.js
function myFunction(param) {
    let x = param;
    x += 10;
    if (x > 15) {
        x -= 5;
    } else {
        x += 5;
    }
    for (let i = 0; i < 3; i++) {
        x += i;
    }
    return x;
}

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question('Enter an integer: ', input => {
    const param = parseInt(input, 10);
    if (isNaN(param)) {
        console.log('Invalid input');
    } else {
        console.log(myFunction(param));
    }
    readline.close();
});

File: function.kt
// function.kt
fun myFunction(param: Int): Int {
    var x = param
    x += 10
    if (x > 15) {
        x -= 5
    } else {
        x += 5
    }
    for (i in 0..2) {
        x += i
    }
    return x
}

fun main() {
    print("Enter an integer: ")
    val input = readLine()
    val param = input?.toIntOrNull()
    if (param == null) {
        println("Invalid input")
    } else {
        println(myFunction(param))
    }
}
